contribute.html

```diff
@@ -1,3 +1,9 @@
+---
+layout: blank
```

Comment:
is the blank layout needed?

Reply:
Yeah, unfortunately. Otherwise, it uses the default one which messes up the page (I've tried using `layout: null` too) :(

<img width="1680" alt="Screen Shot 2019-06-24 at 5 10 33 PM" src="https://user-images.githubusercontent.com/42985749/60052343-00df9880-96a3-11e9-983e-9f7af43d1834.png">

Reply:
ah that does not look nice. thanks for clarifying!